user-guide/Troubleshooting/Error_Messages/Failed_to_get_log_file.md

```diff
@@ -0,0 +1,83 @@
+---
+uid: Failed_to_get_log_file
+---
+
+# Failed to get log file
+
+If Cube is unable to load its log files, it will show an error code in the following format:
+
+```txt
+Failed to get log file (<HTTP Error Code> - <ErrorMessage>)
+```
+
+This can have various causes, depending on the error code and message. You can find more detailed information about this below.
+
+## 401 - Unauthorized
+
+```txt
+Failed to get log file (401 - Unauthorized)
+```
+
+This error code and message can have the following possible causes:
+
+- **Possible cause**: Some DLLs and/or services do not have the correct rights.
+
+  **Resolution**: On the server, go to `C:\Skyline DataMiner\Tools` and run *dcomConfig.exe* as administrator.
+
+- **Possible cause**: When HTTPS got configured recently and you get this error, something is likely to be misconfigured. If this is the root cause, you may also get a pop-up message mentioning "Your connection is not private", giving you the option to proceed or to not enter the site. This depends on the type of certificate used.
+
+  **Resolution**: Double-check your HTTPS configuration. The most likely cause is that the *HTTPS* tag in *MaintenanceSettings.xml* was not correctly configured.
+
+  > [!TIP]
+  > See also:
+  >
+  > - [Setting up HTTPS on a DMA](xref:Setting_up_HTTPS_on_a_DMA)
+  > - [HTTPS Configuration BPA](xref:BPA_Https_Configuration)
+
+## 403 - Forbidden
+
+```txt
+Failed to get log file (403 - Forbidden)
+```
+
+This error code and message can have the following possible cause:
+
+- **Possible cause**: This usually occurs when a proxy server is configured. To check if this is indeed the case, open a command prompt on the server and execute the following command:
+
+  ```txt
+  netsh winhttp show proxy
```

Comment:
What should you see when you execute this command? It might be useful to add this, so the user will know what to do based on the result.

Reply:
The result is actually quite straightforward, it either shows "Direct access (no proxy server)." or it shows the proxy server used and the bypass list
![image](https://github.com/SkylineCommunications/dataminer-docs/assets/99317732/192be033-a65e-403a-88b8-359ef90e257f)
In the next command, we update the bypass list